js/testapps/rag/src/index.ts

```diff
@@ -82,8 +84,23 @@ export default configureGenkit({
     ]),
     chroma([
       {
-        collectionName: 'dogfacts_collection',
+        collectionName: 'my_collection',
         embedder: textEmbeddingGecko,
+        createCollectionIfMissing: true,
+        clientParams: async () => {
+          const host = 'https://chroma-974125110527.us-central1.run.app';
+          const authClient = await auth.getIdTokenClient(host);
```

Comment:
this client can be cached outside of the params callback?

Reply:
Done.